src/orchestrator-tracing/src/lib.rs

```diff
@@ -269,6 +269,13 @@ impl NamespacedOrchestrator for NamespacedTracingOrchestrator {
             }
             #[cfg(feature = "tokio-console")]
             if let Some(port) = tokio_console_port {
+                tracing::info!(
+                    "{}:{} starting tokio console on port: {}",
+                    id,
+                    assigned.index.unwrap_or(0),
+                    port
+                );
```

Comment:
Wait, this seems real spammy! Why do we need to log it in both places? In Kubernetes this is going to print something like:

```
computed:0 starting tokio console on port 6667
computed:0 starting tokio console on port 6667
computed:0 starting tokio console on port 6667
computed:0 starting tokio console on port 6667
computed:0 starting tokio console on port 6667
```

which does not seem very helpful!

Reply:
@benesch its to improve the naming; for example, when we have 1 storaged-per-source, is that storaged going to be started with enough information in its flags to print out a meaningful log line? my understanding that the process will only get the name of the source its starting as a command later on

Reply:
Ohh, I see! ISTM like the `--log-include-service-name` should probably change to a `--log-prefix` option that includes more than the service name! Like it'd be swell if _all_ of the log messages produced by a `computed` process or a `storaged` process were actually prefixed with e.g. `storage-s12` or `computed-cluster-0-replica-1`!

Reply:
Oh hmm, thats a great point, let me break that out into a separate pr!